huh_test.go

```diff
@@ -936,6 +936,150 @@ func TestGetFocusedField(t *testing.T) {
 	}
 }
 
+func TestTitleRowRender(t *testing.T) {
+	t.Run("Confirm", func(t *testing.T) {
+		withTitle := NewConfirm().Title("A Title")
+		noTitle := NewConfirm().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+
+	t.Run("FilePicker", func(t *testing.T) {
+		withTitle := NewFilePicker().Title("A Title")
+		noTitle := NewFilePicker().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+
+	t.Run("Input", func(t *testing.T) {
+		withTitle := NewInput().Title("A Title")
+		noTitle := NewInput().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+
+	t.Run("MultiSelect[string]", func(t *testing.T) {
+		withTitle := NewMultiSelect[string]().Title("A Title")
+		noTitle := NewMultiSelect[string]().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+
+	t.Run("Note", func(t *testing.T) {
+		withTitle := NewNote().Title("A Title")
+		noTitle := NewNote().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+
+	t.Run("Select[string]", func(t *testing.T) {
+		withTitle := NewSelect[string]().Title("A Title")
+		noTitle := NewSelect[string]().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+
+	t.Run("Text", func(t *testing.T) {
+		withTitle := NewText().Title("A Title")
+		noTitle := NewText().Title("")
+		got := lipgloss.Height(ansi.Strip(noTitle.View()))
+		want := lipgloss.Height(ansi.Strip(withTitle.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
+	})
+}
+
+func TestDescriptionRowRender(t *testing.T) {
+	t.Run("Confirm", func(t *testing.T) {
+		withDescription := NewConfirm().Description("A Description")
+		noDescription := NewConfirm().Description("")
+		got := lipgloss.Height(ansi.Strip(noDescription.View()))
+		want := lipgloss.Height(ansi.Strip(withDescription.View())) - 1
+		if got != want {
+			t.Fatalf("got != want; height should be %d, got %d", want, got)
+		}
```

Comment:
would probably be better to do something like:

```suggestion
		c := NewConfirm().Description("A Description")
		if view := c.View(); !strings.Contains(view, "A Description") {
			t.Log(pretty.Render(view))
			t.Error("Expected description to be visible")
		}
```

on all these tests

on a second thought, this can be a bit less repetitive as well, let me do it in #548 as I'm already fixing things related to titles/descriptions there 🙏

Reply:
@caarlos0 the only thing is that when there are empty titles, the row still renders, then when the field is selected, it accentuates that

![image](https://github.com/user-attachments/assets/e52aec16-6754-4044-9603-096571543f0b)

In essence what I'm trying to test here is that the field height shrinks if the description and titles are not set